source/common/json/config_schemas.cc

```diff
@@ -156,7 +156,8 @@ const std::string Json::Schema::LISTENER_SCHEMA(R"EOF(
             }
           },
           "cipher_suites" : {"type" : "string", "minLength" : 1},
-          "ecdh_curves" : {"type" : "string", "minLength" : 1}
+          "ecdh_curves" : {"type" : "string", "minLength" : 1},
+          "crl_file" : {"type" : "string"}
```

Comment:
v1 APIs are deprecated, so I'm not sure if we're supposed to keep adding things here? cc @mattklein123 @htuch

Reply:
At this point v2 is required, v1 is optional but not blocked. If folks want to add and doc no problem.